src/domain/prompt/AdvancedSelectOnePrompt.ts

```diff
@@ -17,13 +17,14 @@
  * OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  **/
 
-import {assertNotNull, BasePrompt, IAdvancedSelectOne, IAdvancedSelectOnePromptConfig, ValidationException} from '../..'
+import {BasePrompt, IAdvancedSelectOnePromptConfig} from '../..'
 
 export const ADVANCED_SELECT_ONE_PROMPT_KEY = 'AdvancedSelectOne'
 
 export class AdvancedSelectOnePrompt extends BasePrompt<IAdvancedSelectOnePromptConfig> {
-  validate(selectedRow?: IAdvancedSelectOne[], choiceRows?: string[][]): boolean {
-    const {choiceRowFields, isResponseRequired} = this.config
+  validate(): boolean {
+    // TODO: Decide what to do with this after discussing with Brett
```

Comment:
Resolve?

Reply:
This is a revert of Roan's changes since the interface is now different. 

Not sure what should happen from here on @AlteaDown

Reply:
Yeah, so this is commented out, due to this data existing in a downloaded CSV file, that the flowRunner does not have access to, as it's on whatever downloaded the file. If we wanted to make this work, we would need to provide the Prompt with an interface it can use to use file.